cmd/run/run.go

```diff
@@ -102,6 +102,10 @@ func (c *Command) entrypoint(ctx context.Context, args []string) error {
 		return nil
 	}
 
+	if val := os.Getenv("SUBTRACE_TOKEN"); val == "" {
```

Comment:
What does SUBTRACE_TOKEN represent?

Reply:
It's the Subtrace API token users will generate from the dashboard for their project. It's sent on all API requests to the control server in the `authorization` header.